src/reducers/initialState.js

```diff
@@ -0,0 +1,8 @@
+const initialState = {
```

Comment:
call this initialUser or anything else because we shall have different initial states here